build.gradle

```diff
@@ -195,18 +201,21 @@ repositories {
 
 processResources.dependsOn(generateGitProperties)
 
-project.tasks['sonarqube'].dependsOn test, integration, jacocoTestReport
+project.tasks.named('sonar') {
```

Comment:
is there a reason we're changing the task name?

Reply:
sonarqube came up as a deprecated task in gradle